system_tests/datastore.py

```diff
@@ -35,8 +42,17 @@ class Config(object):
 
 
 def setUpModule():
-    client.DATASET = TESTS_DATASET
-    Config.CLIENT = datastore.Client()
+    if EMULATOR_DATASET is None:
+        client.DATASET = TESTS_DATASET
+        Config.CLIENT = datastore.Client()
+    else:
+        Config.CLIENT = datastore.Client(dataset_id=EMULATOR_DATASET)
+        populate_datastore.add_characters(client=Config.CLIENT)
```

Comment:
I elected to pass an explicit `dataset_id` to the constructor instead (to avoid side-effects)